runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragConfig.java

```diff
@@ -40,6 +40,24 @@
 	)
 	default int dragDelay()
 	{
-		return Constants.GAME_TICK_LENGTH / Constants.CLIENT_TICK_LENGTH; // one game tick
+		return Constants.GAME_TICK_LENGTH / Constants.CLIENT_TICK_LENGTH;
+	}
+
+	@ConfigItem(
+		keyName = "dragDelay",
+		name = "",
+		description = ""
+	)
+	void dragDelay(int delay);
```

Comment:
I can't figure out what this is here for. I don't think it's being used?

Reply:
This was an accident. I have removed it.

Reply:
This still has not been removed. Why was this comment marked resolved?

Reply:
Sorry. I did remove this error, but someone accidentally re-pushed it back out.